linux_os/guide/services/ssh/ssh_server/sshd_disable_root_password_login/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Disable SSH Root Login with a password'
+
+description: |-
+    The root user should not be allowed to login to a
+    system directly over a network with a password.
+    To disable root login via SSH, add or correct the following line
+    in <tt>/etc/ssh/sshd_config</tt>:
+    <pre>PermitRootLogin prohibit-password</pre>
```

Comment:
The first and second sentences need some clarification. While the first is about password-based logins, the second sentence only mentions disabling root over SSH.  

Would something like this work?

`````
To disable password-based root logins over SSH, add or correct the following line
in <tt>/etc/ssh/sshd_config</tt>:
<pre>PermitRootLogin prohibit-password</pre>

NOTE: While this disables password-based root logins, direct root logins through other means (eg SSH keys) will still be permitted. To fully disable direct root logins over SSH, see CCE-XXXX: Disable root Logins
`````

Reply:
I do like this, I'll add in the GSSAPI bits and format to 80 col.